test/util/Util.ts

```diff
@@ -168,3 +169,15 @@ export const mockFs = (rootFilepath?: string, time?: Date): { data: any } => {
 
   return cache;
 };
+
+/** Mocks an HTTP response. */
+export const mockHttpResponse = (): any => {
+  const headers: any = {};
+  return {
+    headers,
+    hasHeader: (name: string): boolean => name in headers,
+    setHeader(name: string, value: string): void {
+      headers[name] = value;
+    },
+  } as any as HttpResponse;
+};
```

Comment:
We actually already use the `node-mocks-http` library in some places (`BasicResponseWriter` tests for example). So I would reuse that one instead of making this function then.

Reply:
Check, will do.